test/fixtures/lightscript/if-statement/oneline-return-undelimited/actual.js

```diff
@@ -0,0 +1,2 @@
+f = ->
+  if (true) return 1 else return 2
```

Comment:
yeah, sorry, again this was my bad – let's test the other forms here:
```
if (true) return 1 
else return 2
```
and
```
if (true) 1 else 2
```
both of which should be legal, if discouraged. 

fine to leave this test too